src/main/g8/src/main/scala/$package$/UserRoutes.scala

```diff
@@ -1,90 +1,74 @@
 package $package$
 
-import akka.actor.{ ActorRef, ActorSystem }
-import akka.event.Logging
-
 import scala.concurrent.duration._
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.server.Route
-import akka.http.scaladsl.server.directives.MethodDirectives.delete
-import akka.http.scaladsl.server.directives.MethodDirectives.get
-import akka.http.scaladsl.server.directives.MethodDirectives.post
-import akka.http.scaladsl.server.directives.RouteDirectives.complete
-import akka.http.scaladsl.server.directives.PathDirectives.path
 
 import scala.concurrent.Future
-import $package$.UserRegistryActor._
-import akka.pattern.ask
+import $package$.UserRegistry._
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.AskPattern._
 import akka.util.Timeout
 
+//#import-json-formats
 //#user-routes-class
-trait UserRoutes extends JsonSupport {
+class UserRoutes(userRegistry: ActorRef[UserRegistry.Command])(implicit val system: ActorSystem[_]) {
```

Comment:
Not doing these dependencies mixin-style saves us from making things depending on them lazy below just to not break because of initialization order.